### ISSUE TITLE
An Error Occurred After An Update

### ISSUE THREAD

**Issue by ProfessorCode212** [tutao/tutanota#3160]
- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Describe the bug**
I got a notification with an error message saying something like _Failed to handle PushMessage_ after updating Tutanota to version 3.84.4. On trying to login afterwards, I received another error.
```
Feedback v3.84.4 - TypeError - UNKNOWN - linux



 Client: linux
 Type: UNKNOWN
 Tutanota version: 3.84.4
 Timestamp (UTC): Thu, 03 Jun 2021 22:37:06 GMT
 User agent:
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) tutanota-desktop/3.84.4 Chrome/87.0.4280.141 Electron/11.4.6 Safari/537.36
TypeError
 Error message: Cannot read property 'userGroup' of null
Stacktrace: 
TypeError: Cannot read property 'userGroup' of null
    at wa.getUserGroupId (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/worker.js:1:107952)
    at wa.getUserGroupKey (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/worker.js:1:108146)
    at wa._initIndexer (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/worker.js:1:106451)
    at file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/worker.js:1:106845
    at l (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/polyfill.js:1:81787)
    at H._settlePromiseFromHandler (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/polyfill.js:1:54790)
    at H._settlePromise (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/polyfill.js:1:55590)
    at H._settlePromise0 (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/polyfill.js:1:56289)
    at H._settlePromises (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/polyfill.js:1:57641)
    at H._fulfill (file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/polyfill.js:1:56659)
    at file:///tmp/.mount_tutanoyCLMVA/resources/app.asar/polyfill.js:1:77762
```
It was only after I closed Tutanota and restarted the application that I successfully logged in to my account.

**Desktop (please complete the following information):**
 - OS: Linux Mint 19.3
 - Desktop Environment: Cinnamon 4.4.8

**Comment by johnbotris** [comment]
Duplicate of #3050
